addons/portal/static/src/scss/portal.scss

```diff
@@ -382,6 +382,10 @@ img, .media_iframe_video, .o_image {
         border: 0;
     }
 
+    .o_portal_chatter_composer_body > div {
```

Comment:
Would be a bit better if we merged rules like this:

```scss
.o_portal_chatter_composer_body  {
    textarea {
        border: 0;
    }
    > div {
        border: 1px solid var(--o-border-color);
    }
}
```

Reply:
Hello @awa-odoo 

Changes are done can you please review it again?
Thanks :blush: